src/main/java/salesforce/lightning/WorkTypeLightningPage.java

```diff
@@ -1,15 +1,30 @@
 package salesforce.lightning;
 
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
 import salesforce.BasePage;
 
 public class WorkTypeLightningPage extends BasePage {
+    @FindBy(xpath = "//a[@class='forceActionLink'][@role='button']")
+    protected WebElement newBtn;
 
     public WorkTypeLightningPage(final WebDriver driver) {
         super(driver);
```

Comment:
you should pass the driver, since we are using singleton and it can be called from anywhere, review that.